ingestion/src/metadata/ingestion/source/mlmodel/sagemaker/metadata.py

```diff
@@ -151,6 +151,22 @@ def yield_mlmodel(  # pylint: disable=arguments-differ
                 )
             )
 
+    def _get_image_repository(self, model_name: str, model_info: dict) -> Optional[str]:
+        try:
+            return model_info["PrimaryContainer"]["Image"]
```

Comment:
should we define pydantic models to convert AWS SDK dict payload into what we expect in one go?